source/sdk/threading/Thread.ooc

```diff
@@ -165,6 +172,23 @@ Mutex: abstract class {
 
 }
 
+MutexUnsafe: class extends Mutex {
+	init: func
+	lock: override func
```

Comment:
I'll just leave this here:
```ooc
Foo: abstract class {
    init: func
    hey: abstract func
}
Bar: class extends Foo {
    init: func
    hey: override func { println("hey") }
}
Baz: class extends Bar {
    init: func
    hey: override func { println("hey you") } // won't work, because `hey` in `Bar` isn't virtual
}
```
and
```ooc
Foo: abstract class {
    init: func
    hey: abstract func
}
Bar: class extends Foo {
    init: func
    hey: func { println("hey") }
}
Baz: class extends Bar {
    init: func
    hey: override func { println("hey you") }

bar := Bar new()
bar hey()
baz := Baz new()
baz hey()
```
both print
```
hey
hey you
```
but
```ooc
Foo: abstract class {
    init: func
    hey: abstract func
}
Bar: class extends Foo {
    init: func
    hey: virtual func { println("hey") }
}
Baz: class extends Bar {
    init: func
    hey: override func { println("hey you") }
}

bar := Bar new()
bar hey()
baz := Baz new()
baz hey()
```
prints
```
Child received signal 11 (Segmentation fault)
```
Perhaps @fredrikbryntesson can explain why the `override` keyword in `Bar` is optional?